one_installer.yml

```diff
@@ -9,88 +8,208 @@
   tasks:
     - name: Update package repository
       shell: sudo apt update
+      ignore_errors: yes
```

Comment:
what is this rule doing?

Reply:
On updating packages,errors come up if there are deprecated dependencies.Then it won't go to the next task.So what ignore_errors does is even if an error comes up,t won't stop the installation of the scripts which comes after this task